codeship-steps.yml

```diff
@@ -6,6 +6,8 @@
   command: true
 - service: integrationtest
   command: deploy_to_heroku_app codeship-heroku-deployment
+- service: integrationtest
+  command: heroku run --app codeship-heroku-deployment -- ls
```

Comment:
hmmmm why is this not in there. Thought I changed it. I'll change to true and merge after the build works